cmd/rokka/cli/sourceimages.go

```diff
@@ -68,6 +70,38 @@ func restoreSourceImage(c *rokka.Client, args []string) (interface{}, error) {
 
 func copySourceImage(c *rokka.Client, args []string) (interface{}, error) {
 	return nil, c.CopySourceImage(args[0], args[1], args[2])
+
+}
+
+func copyAllSourceImage(c *rokka.Client, args []string) (interface{}, error) {
+
+	images := make(chan string)
+	results := make(chan copyall.CopyResult)
+	quit := make(chan bool)
```

Comment:
The quit channel is not used (you never send anything to it). 
Now that I think about that, I think the quit channel in #60 is unnecessary too, because `ctrl+c` will do it anyway.

Reply:
yep. removed it